handlers/handlers.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: restart mesos-slave
+  service:
+    name: mesos-slave
+    enabled: true
+    state: restarted
+
+- name: stop mesos-slave jobs on master
```

Comment:
Please rename this handler to ``stop mesos-slave``. The reason why I am suggesting this is:
* To keep it uniform to all handlers
* Here we are not stopping mesos-slave jobs we are disable mesos-slave daemon on masters